### ISSUE TITLE
Send Payment button is misleading for PayPalPaymentIntentAuthorize

### ISSUE THREAD

**Issue by SinisterCB** [paypal/PayPal-iOS-SDK#174]
Our app uses the PayPal iOS SDK for payment authorization, and the final payment capture occurs only after the customer approves the final order details. We create a PayPalPayment with .intent == PayPalPaymentIntentAuthorize. However, in the PayPalPaymentViewController, there is no adjustment for the title of the "Send Payment" button. In this case, the button should reflect that the customer is "authorizing" the payment, but not actually "sending" it.

**Comment by keenan16** [comment]
Hi, How to void transaction on the app side?

**Comment by dgoldman-ebay** [comment]
@keenan16 your question is unrelated to this issue. Please open a new issue instead. And when you do, could you also please provide more details regarding what you're trying to accomplish, and why?

Thanks!

**Comment by keenan16** [comment]
Hi @dgoldman-ebay sorry for misleading the topic. The set up of @SinisterCB is also the same with us.
here is my details of my question, https://github.com/paypal/PayPal-iOS-SDK/issues/253

Thank you very much.

**Comment by jasonblood** [comment]
:+1:  Would also like to see this done its rather confusing when authorising payments because the interface makes it looks like the payment is taking place on the client

**Comment by dgoldman-ebay** [comment]
@SinisterCB and @jasonblood:

So, apparently the standard PayPal terminology for this sort of button is one of the following; `Pay`, `Pay Now`, or `Continue`.

If we switched from the current `Send Payment` to simply `Pay`, do you think that would be less confusing?

**Comment by SinisterCB** [comment]
Pay is less confusing than Send Payment because to me it allows for more
ambiguity in the timing (doesn't sound as immediate).

However, for clarity let me describe our use case -- we were trying to
achieve this flow:
1. create project and details of order
2. provide shipping information
3. provide billing information (address, phone, etc.)
4. provide payment information (credit card, paypal, etc.)
5. confirm final details of order and place the order

Payment AUTHORIZATION occurs at step 4 - we take your credit card number
directly if you wish to provide it and we authorize it via Stripe. We also
wished to be able to authorize (but not transact) a PayPal transaction at
the customer's choice.

Payment EXECUTION happens after step 5 - in the case of a credit card, we
capture the payment that was previously authorized. In the case of PayPal
we would do the same.

However, because the authorize/execute difference isn't so clear in PayPal,
we ended up skipping the authorization at step 4 - PayPal becomes a choice
selection at step 4 but a no-op until after step 5 at which point we go
ahead and capture the payment by showing the UI.

Therefore, for us Pay is still not the ideal terminology to present to the
customer at step 4.  Authorize would be clearer, especially since we may
not end up capturing the payment if the customer decides not to proceed
once he/shes see the full confirmation (price, tax, shipping, discounts,
etc. all reflected together). We do see a small number of credit card
authorizations that do not result in transactions - presumably this would
be the same in the PayPal case.

For our needs, it would be ideal to offer either (a) a list of constants
that relate to possible localized string options for the button, the
constant to be specified as part of the API call, or (b) the option for a
client-specified string offered as a parameter to the API call.

**Comment by dgoldman-ebay** [comment]
1. In the [latest release of the PayPal-iOS-SDK](https://github.com/paypal/PayPal-iOS-SDK/releases), this button now says `Pay` rather than `Send Payment`.

2. @SinisterCB I appreciate your thorough explanation! If we stick to standard PayPal terminology here (as I think is going to have to be the case), then would `Continue` be better for you? If so, maybe I can eventually get internal agreement to allow apps to tell our SDK whether to use `Pay` or `Continue`.
